scripts/install.go

```diff
@@ -0,0 +1,62 @@
+package scripts
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+//go:embed setup/setup.sh
+//go:embed setup/config-template.toml
+//go:embed setup/coriolis-snapshot-agent.service.sample
+var setupContent embed.FS
+
+const (
+	setupTmpDir = "/tmp/setup"
+)
+
+func writeEmbeddedFile(filename string, mode fs.FileMode) error {
+	fileContent, err := setupContent.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+
```

Comment:
You'll need to ensure the base dir exists:

```suggestion
filePath := filepath.Join(dir, filename)
dirName := filepath.Dir(filePath)
if _, err := os.Stat(dirName); err != nil {
    if !os.IsNotExist(err) {
        return err
    }
    if err := os.MkdirAll(dirName, 0750)
        return err
    }
}
```